<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="fileList">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>841</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Action And events</string>
   </property>
   <widget class="QTableWidget" name="detailTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>651</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Discription</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priorities</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Action on</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Action to Do</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>image</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Enabled</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Pixel Bot</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>691</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Logs</string>
   </property>
   <widget class="QTextEdit" name="logs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Create New File</string>
   </property>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="craeteFileInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
